flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/BatchTableEnvironment.scala

```diff
@@ -228,19 +229,13 @@ abstract class BatchTableEnvironment(
   }
 
   /**
-    * Translates a [[Table]] into a [[DataSet]].
-    *
-    * The transformation involves optimizing the relational expression tree as defined by
+    * Generates the optimized [[RelNode]] tree from the relational expression tree as defined by
     * Table API calls and / or SQL queries and generating corresponding [[DataSet]] operators.
```

Comment:
The method does not generate a `DataSet` operators. Can you also add that it converts to DataSet/DataStream convertion?